lib/pr_checker.js

```diff
@@ -404,6 +404,14 @@ export default class PRChecker {
       return false;
     }
 
+    const files = pr.files.nodes;
+
+    // Don't require Jenkins run if pr is doc-only changes
+    const isDocOnlyChange = files.every(({ path }) => path.endsWith('.md'));
+    if (isDocOnlyChange) {
```

Comment:
I don't think we need a variable that we don't reuse later, and the comment above already explains what it does.
```suggestion
    if (files.every(({ path }) => path.endsWith('.md'))) {
```